wgengine/magicsock/magicsock.go

```diff
@@ -3027,7 +3033,7 @@ func (c *Conn) bindSocket(ruc *RebindingUDPConn, network string, curPortFate cur
 		return nil
 	}
 
-	if debugAlwaysDERP() {
+	if debugAlwaysDERP() || debugBindSocket() {
```

Comment:
this seems like it's changing behavior rather than just adding debug prints

Reply:
Blergh, good catch; fixed.